### PR TITLE
Feature/music generation

### PR DESCRIPTION
**Pull request by bartoszrud**
Add generation of music based on the description using [MusicGen model](https://huggingface.co/docs/transformers/main/en/model_doc/musicgen).

### CODE REVIEW

File: src/typehints/transformers/models.pyi [hunk 1]
```diff
@@ -12,3 +12,34 @@ class AutoTokenizer:
...
     def __call__(self, text: str, return_tensors: str) -> torch.Tensor: ...
+
+class PretrainedConfig:
+    def __init__(self, **kwargs): ...
+
+class EncodecConfig(PretrainedConfig):
+    sampling_rate: int
+
+class MusicgenDecoderConfig(PretrainedConfig):
+    pass
```
Comment (BartWojtowicz) [inline src/typehints/transformers/models.pyi:23]: I don't like a little doing this `pass` here, would it work with just `...` instead? 
```suggestion
class MusicgenDecoderConfig(PretrainedConfig):
    ...
```

**Review by BartWojtowicz — COMMENTED** [review]
Looking good 🥰 

Please add also a small usage example in the generation section of our README

File: src/typehints/transformers/models.pyi (hunk 1 above)
Comment (bartoszrud) [reply src/typehints/transformers/models.pyi]: Good point, done!

**Comment by bartoszrud** [comment]
> Looking good 🥰
> 
> Please add also a small usage example in the generation section of our README

While trying to add some usage example I realised that we cannot add audio directly to the video. We can add it only from file. I've created [an issue](https://github.com/BartWojtowicz/videopython/issues/85) for that.

**Review by BartWojtowicz — APPROVED** [review]
Looking great 🥰

**Pull request merged**